tools/kcli/tests/test_yaml.py

```diff
@@ -15,3 +16,15 @@ def test_unsupported_yaml_constructor(our_cwd_setup):
     settings = configure.Configuration.from_dict({})
     with pytest.raises(IRYAMLConstructorError):
         update_settings(settings, os.path.join(utils.TESTS_CWD, tester_file))
+
+
+def test_placeholder_validator(our_cwd_setup):
+    from kcli.utils import update_settings
+    from kcli.exceptions import IRPlaceholderException
+    tester_file = 'placeholder_validator.yml'
+    settings = configure.Configuration.from_dict({})
+    with pytest.raises(IRPlaceholderException) as exc:
+        settings = update_settings(settings,
+                                   os.path.join(utils.TESTS_CWD, tester_file))
+        yaml.safe_dump(settings, default_flow_style=False)
+    assert "Mandatory value is missing." in str(exc.value.message)
```

Comment:
Please add unittest that a 2nd yaml can overwrite the place holder and avoid the exception